sources/web/datalab/templates/edit.html

```diff
@@ -28,43 +28,47 @@
         <a href="/tree"><img src="/static/logo.png" alt="Google Cloud DataLab" /></a>
       </span>
       <span id="save_widget" class="pull-left save_widget">
-        <button class="btn" title="Rename file"><span id="file_name" class="filename"></span></button>
+        <button class="toolbar-btn" title="Rename file"><span id="file_name" class="filename"></span></button>
         <span class="autosave_status"></span>
       </span>
       <div class="btn-toolbar pull-right">
         <div class="btn-group">
-          <button id="feedbackButton" title="Provide feedback">
+          <button id="feedbackButton" title="Provide feedback" class="toolbar-btn">
             <span class="fa fa-comment"></span>
           </button>
         </div>
         <div class="btn-group">
-          <button id="aboutButton" title="About Google Cloud Datalab">
+          <button id="aboutButton" title="About Google Cloud Datalab" class="toolbar-btn">
             <span class="fa fa-info-circle"></span>
           </button>
         </div>
         <div class="btn-group">
-          <a href="/tree/datalab/docs/notebooks" target="_blank" id="docsButton" class="btn">
-            <button id="docsButton" title="Samples and Tutorials">
+          <a href="/tree/datalab/docs/notebooks" target="_blank" id="docsButton">
+            <button id="docsButton" title="Samples and Tutorials" class="toolbar-btn">
               <span class="fa fa-book"></span>
             </button>
           </a>
         </div>
         <div class="btn-group">
-          <a href="/sessions" target="_blank" id="sessionsButton" class="btn">
-            <button title="Running Sessions">
+          <a href="/sessions" target="_blank" id="sessionsButton">
+            <button title="Running Sessions" class="toolbar-btn">
               <span class="fa fa-tasks"></span>
             </button>
           </a>
         </div>
         <div class="btn-group">
-          <button id="signInButton" title="Sign In" style="display:none">
-            <span class="fa fa-sign-in"></span>
-          </button>
-        </div>
-        <div class="btn-group">
-          <button id="signOutButton" title="Sign Out (<%account%>)" style="display:none">
-            <span class="fa fa-sign-out"></span>
-          </button>
+          <button id="signInButton" title="Sign In" style="display:none" class="toolbar-btn">Sign In</button>
+          <div id="signOutGroup" style="display:none">
+            <button title="Account" data-toggle="dropdown" class="toolbar-btn">
+              <span class="fa fa-user"></span>
+            </button>
+            <ul class="dropdown-menu dropdown-menu-right" style="padding: 10px">
+              <li>
+                <label style="color: #000">Signed in as <%account%></label>
```

Comment:
> style="color: #000"

I'd prefer that we add an ID to these elements (e.g. 'id="signedInAccount"'), and then style it in the css files.

Otherwise, we'll have difficulty modifying this for the dark theme so that it is readable.